specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/dynatrace.json

```diff
@@ -0,0 +1,2244 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Dynatrace.Observability",
+    "version": "2023-04-27"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Monitors"
+    },
+    {
+      "name": "Operations"
+    },
+    {
+      "name": "TagRules"
+    },
+    {
+      "name": "SingleSignOn"
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Dynatrace.Observability/monitors/{monitorName}/listMonitoredResources": {
```

Comment:
>Dynatrace.Observability [](http://example.com/codeflow?start=82&length=23)

Just curious on the naming for this RP, it is different from other Microsoft.* named RPs, I've seen other Liftr RPs like datadog and elastic use Microsoft.* naming convention. Why is it different from them?

Reply:
2 years ago, when the project started PM suggested a new pattern for observability partners. Hence, they come up with Observability suffix